src/routes/Auth.vue

```diff
@@ -34,18 +40,23 @@ export default {
     },
     mounted() {
         const uiConfig = {
-            signInSuccessUrl: '/game',
+            signInSuccessUrl: '/',
             signInOptions: [
                 firebase.auth.GoogleAuthProvider.PROVIDER_ID,
                 firebase.auth.EmailAuthProvider.PROVIDER_ID
                 ]
         };
-        let ui = new firebaseui.auth.AuthUI(firebase.auth());
+        let ui = firebaseui.auth.AuthUI.getInstance();
+        if (!ui) {
+            ui = new firebaseui.auth.AuthUI(firebase.auth());
+        }
         ui.start('#firebaseui-auth-container', uiConfig);
 
     },
   methods: {
-
+      signOut(){
+        firebase.auth().signOut();
```

Comment:
Is this assuming that `firebase` is necessarily non-`null`, and does this assertion hold true always?

Reply:
I'm not entirely sure that this will be 100% the case, but when the app launches it should initialize firebase, and if not the vue instance probably won't even render, so I'm guessing it is safe to assume firebase would be non-null here?

Reply:
Then it might be a good idea to either:
- Hard-enforce this by making it do some `console.error` or abort; or
- Make this a no-op in case `firebase` is not initialized, i.e. `firebase?.auth()?.signOut()`;

Just letting this silently fail is hard to debug, just IMO.

Reply:
Sure I'll add a check